Source/WebCore/contentextensions/ContentExtensionParser.h

```diff
@@ -37,10 +37,13 @@ namespace WebCore {
 namespace ContentExtensions {
 
 class ContentExtensionRule;
+struct CSSParserContext;
```

Comment:
This is incorrect.  It should be inside namespace WebCore but not inside namespace ContentExtensions.